lib/base64.nit

```diff
@@ -77,62 +77,74 @@ redef class NativeString
 
 	# Decodes `self` from base64
 	#
+	# `strict` is the reverse-equivalent of the `-i` option of
+	# the `base64` Unix command.
+	# i.e., it will check if all characters in `self` are from the
+	# `base64` alphabet and is correctly padded if needed.
+	# `strict` defaults to `true`
+	#
 	#      assert "c3RyaW5n".decode_base64 == "string"
 	#      assert "c3Rya\nW5n".decode_base64 == "string"
+	#      assert "c3RyaW5nCg==".decode_base64 == "string\n"
+	#      assert "c3RyaW5nCg".decode_base64(strict=false) == "string\n"
+	#      assert "c3RyaW5neQo=".decode_base64 == "stringy\n"
+	#      assert "c3RyaW5neQo".decode_base64(strict=false) == "stringy\n"
 	#
-	# REQUIRE: `length % 4 == 0`
-	private fun decode_base64(length: Int, padding: nullable Byte): Bytes do
-		if padding == null then padding = '='.ascii
+	private fun decode_base64(length: Int, padding: nullable Byte, strict: nullable Bool): Bytes do
```

Comment:
Seriously, I just do not get the point of a flag that says: we could easily have a fallback/loose semantic, but we just prefer to crash your program without any sane recovering way.